Edits(?) & fixes (#183)
shawntabrizi/substrate-collectables-workshop: steps/52/README.md

In this context, we don't want someone to kill their account to buy a kitty, so we want to use `Preservation::Preserve` for our `transfer`.

> > NOTE: To access this enum, you will need import the trait to bring it in scope, so don't forget to include:

Reviewer: ```suggestion
> NOTE: Don't forget the TODO that imports this enum so you can use it:
```
